.circleci/config.yml

```diff
@@ -1234,6 +1234,10 @@ workflows:
           name: op-e2e-ext-geth-tests
           module: op-e2e
           target: test-external-geth
+      - go-e2e-test:
+          name: op-e2e-span-batch-tests
+          module: op-e2e
+          target: test-span-batch
```

Comment:
That's a great point. It would be better if we can identify test cases that we need to run with span batches. But I'm not sure if we can identify them correctly. For instance, I hadn't anticipated that the fault-proof related tests would be impacted by these changes, but I discovered the problem thanks to the new test workflow.
Of course this is my perspective. Since you have a deeper understanding of this system and tests than I do, I think we might be able to group test cases that we need. What do you think?

Reply:
Yeah I can see that it's useful to be able to run all tests to help us find this kind of thing, but it's very expensive to always run all tests with span batches on every run. And we should be able test compatibility of the fault proof op-program by running in natively (eg the `system_fpp_test` cases) rather than in cannon (as in `faultproof_test` which is _way_ slower).

Reply:
Yes I agree with you! let me try to identify tests for span batches.

Reply:
Opened https://github.com/ethereum-optimism/optimism/issues/8022 to track this. I'm ok with merging this with the CI addition. We can address it separate from this PR.